theme/README.md

```diff
@@ -36,10 +36,29 @@ Ruby's official site has a clear [instruction page](https://www.ruby-lang.org/en
 -   `npm run format` to format files.
 -   `npm run zip` to zip the theme files, which can be imported into Zendesk.
 
+## Release Theme on Zendesk
+
+There are two ways to release a new theme on Zendesk. Either by uploading a zipped theme or import the theme via github.
```

Comment:
Wordsmith.

`Either by uploading a zipped theme or by importing the theme via github.`